client/my-sites/theme/main.jsx

```diff
@@ -228,24 +228,6 @@ class ThemeSheet extends React.Component {
 		return demo_uri && ! retired;
 	}
 
-	renderPreviewButton() {
-		return (
-			<a
-				className="theme__sheet-preview-link"
-				onClick={ this.previewAction }
-				data-tip-target="theme-sheet-preview"
-				href={ this.props.demo_uri }
-				rel="noopener noreferrer"
-			>
-				<span className="theme__sheet-preview-link-text">
-					{ i18n.translate( 'Open Live Demo', {
-						context: 'Individual theme live preview button',
-					} ) }
-				</span>
-			</a>
-		);
-	}
```

Comment:
This "Open Live Demo" button seems to be a part of intentional UI design that looks differently on wide two-column layout and narrow, mobile one-column layout
- two-column layout shows an "Open Live Demo" item in a `SectionNav` in the left column
- that item is hidden in one-column view, and the "Open Live Demo" button (the element being removed in this PR) is displayed under the screenshot

@folletto may provide some advice about how to remove the invalid HTML while keeping the design working. He originally worked on it and submitted PRs like #13598.

Also, removing the button breaks the `theme-sheet-welcome-tour` guided tour that looks for the `data-tip-target="theme-sheet-preview"` element and attaches a bubble to it. That tour is likely already broken on non-mobile views (?!)

Reply:
To see this code in action, open a theme page like https://wordpress.com/theme/modern-business and resize the window to switch between one- and two-column views.

Reply:
Ah, so that's why it was there. We should include a comment in the fix so it's more apparent.